core/eolearn/tests/test_utils/test_fs.py

```diff
@@ -135,6 +136,30 @@ def test_tempfs_serialization():
     assert not unpickled_filesystem.exists("/")
 
 
+@mock_s3
```

Comment:
is this decorator needed despite it being used in the fixture?

Reply:
Good point but it turns out it is. If I remove `@mock_s3` for any test it fails saying that the bucket doesn't exist. But I don't have enough understanding how `moto` works to be able to explain why this is so.